pkg/coreapi/graph/resolvers/functions.go

```diff
@@ -6,12 +6,62 @@ import (
 	"sort"
 	"time"
 
+	"github.com/inngest/inngest/inngest"
 	"github.com/inngest/inngest/pkg/coreapi/graph/models"
 	"github.com/inngest/inngest/pkg/enums"
 	"github.com/inngest/inngest/pkg/execution/state"
 	"github.com/inngest/inngest/pkg/function"
 )
 
+func (r *queryResolver) Functions(ctx context.Context) ([]*models.Function, error) {
+	fns, err := r.APIReadWriter.Functions(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	var functions []*models.Function
+	for _, fn := range fns {
+		id := string(fn.ID)
```

Comment:
There were some go-isms that I tried to do in a nicer way, but found to have issues with references so I emulated what was done in other resolvers. Not super pretty, but it works consistently.